### PR TITLE
[T3A3][W11-A2]Ong Heng Le

### PR DESCRIPTION
**Pull request by initialshl**

### CODE REVIEW

**Comment by initialshl** [comment]
Ready for review.

File: src/seedu/addressbook/data/person/Address.java [hunk 1]
```diff
@@ -25,6 +31,12 @@ public Address(String address, boolean isPrivate) throws IllegalValueException {
         if (!isValidAddress(address)) {
             throw new IllegalValueException(MESSAGE_ADDRESS_CONSTRAINTS);
         }
```
Comment (cmkumar87) [inline src/seedu/addressbook/data/person/Address.java]: Delegate validation to the constituent classes. For exampke Block should check validity of the Block part of the address string.

File: src/seedu/addressbook/data/person/Address.java [hunk 2]
```diff
@@ -8,13 +8,19 @@
...
+    private static final String ADDRESS_SEPARATOR = ", ";
 
     public final String value;
     private boolean isPrivate;
 
+    private Block block;
+    private Street street;
+    private Unit unit;
+    private PostalCode postalCode;
+
```
Comment (cmkumar87) [inline src/seedu/addressbook/data/person/Address.java]: Add accessors instead of handling access using toString.

**Comment by cmkumar87** [comment]
@initialshl  Please review the comments and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Pull request closed without merging**